samples/apps/wayfinder-sample/smtp-server/ui/src/App.css

```
@@ -0,0 +1,395 @@
+*, *::before, *::after { box-sizing: border-box; margin: 0; padding: 0; }
+
+:root {
+  color: #111827;
+  background: #ffffff;
+  font-family:
+    Aptos, "Avenir Next", "SF Pro Text", "Helvetica Neue", ui-sans-serif,
+    system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)